internal/toolkit/toolkit.go

```diff
@@ -248,6 +259,15 @@ func (d *DefaultToolkit) init() {
 	if d.Writer == nil {
 		d.Writer = os.Stdout
 	}
+
+	if d.Delemiter == "" {
+		data := make([]byte, 16) // roughly the same entropy as uuid v4 used in https://github.com/actions/toolkit/blob/b36e70495fbee083eb20f600eafa9091d832577d/packages/core/src/file-command.ts#L28
+		_, err := rand.Read(data)
+		if err != nil {
+			panic(fmt.Errorf("could not generate random delimiter: %w", err))
+		}
+		d.Delemiter = hex.EncodeToString(data)
+	}
```

Comment:
Note for reviewer:

GitHub recommends using random delimiters when writing multi-line strings to environment files for security reasons. See https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings
    
Also: It is not unthinkable that an output contains the word `EOF` which would cause problems.

The `Delimiter` member is initialized in the test setup to keep the unit test deterministic.